extensions/arc/package.nls.json

```diff
@@ -98,10 +98,20 @@
 	"arc.postgres.new.dialog.title": "Deploy a PostgreSQL server group on Azure Arc (preview)",
 	"arc.postgres.settings.section.title": "PostgreSQL server group settings",
 	"arc.postgres.settings.resource.title": "PostgreSQL server group resource settings",
+	"arc.postgres.settings.storage.title": "PostgreSQL server group storage settings",
 	"arc.postgres.server.group.name": "Server group name",
 	"arc.postgres.server.group.name.validation.description": "Server group name must consist of lower case alphanumeric characters or '-', start with a letter, end with an alphanumeric character, and be 10 characters or fewer in length.",
 	"arc.postgres.server.group.workers": "Number of workers",
 	"arc.postgres.server.group.port": "Port",
+	"arc.postgres.server.group.engine.version": "Engine Version",
+	"arc.postgres.server.group.extensions.label": "PostgreSQL extensions to enable",
+	"arc.postgres.server.group.extensions.description": "A comma-separated list of the Postgres extensions that should be loaded on startup. Please refer to the postgres documentation for supported values.",
+	"arc.postgres.server.group.volume.size.backups.label": "Volume Size (Backups)",
+	"arc.postgres.server.group.volume.size.backups.description": "The volume size for the storage classes to be used for backups",
+	"arc.postgres.server.group.volume.size.data.label": "The Volume Size (Data)",
+	"arc.postgres.server.group.volume.size.data.description": "The volume size for the storage classes to be used for data",
+	"arc.postgres.server.group.volume.size.logs.label": "The Volume Size (Logs)",
+	"arc.postgres.server.group.volume.size.logs.description": "The volume size for the storage classes to be used for logs",
```

Comment:
I know the descriptions were copied from the azdata CLI, but I think it's confusing to say "The volume size for the storage classes" when there's really just 1 storage class.  So I would say "the storage class"